baselines/classroom.md

```diff
@@ -79,7 +79,7 @@ To configure the settings for Class Membership:
 2.  Select **Save**.
 
 #### GWS.CLASSROOM.1.2v0.3 Instructions
-1.  For Who can join classes in your domain, select **Classes in your domain only**.
+1.  For Which classes can users in your domain join, select **Classes in your domain only**.
```

Comment:
I think ideally "Which classes can users in your domain join" would be bolded too. Arguably out of scope for this PR though, 1.1 instructions would need to be changed to match too. Up to you if you want to make the change now.

```suggestion
1.  For **Which classes can users in your domain join**, select **Classes in your domain only**.
```